### PR TITLE
Feature accordion

### PR DESCRIPTION
**Pull request by Austral1a**

### CODE REVIEW

**Review by MagroTanya — CHANGES_REQUESTED** [review]

File: src/containers/AccordionContainer/hooks/useAccordionContainerManager.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+interface AccordionContainerResult {
+  findActiveId: (id: number) => boolean;
+  onItemToggle: (id: number) => void;
+}
+
+export const useAccordionContainerManager = (): AccordionContainerResult => {
+  const [activeKeys, setActiveKeys] = useState<ActiveKeys[]>([]);
+
```
[the remark this answers is not in the document]
Comment (Austral1a) [reply src/containers/AccordionContainer/hooks/useAccordionContainerManager.tsx:16]: I used array in order to handle entire accordion items if any item has opened every else items will close.

File: src/components/common/Accordion/components/AccordionItem/AccordionItem.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        onItemToggle();
+      }
+    },
+    [onItemToggle]
+  );
+
+  return (
+    <div className={accordionItemClasses}>
+      <div
+        tabIndex={0}
```
[the remark this answers is not in the document]
Comment (Austral1a) [reply src/components/common/Accordion/components/AccordionItem/AccordionItem.tsx:46]: tabIndex is for accessibility purposes. Particular to be able to switch between HTML page elements by Tab key.

[the remark this answers is not in the document]
Comment (Austral1a) [reply src/components/common/Accordion/components/AccordionItem/AccordionItem.tsx:46]: The value of tabIndex - "0" means that an element can be focused on consistently switching between HTML elements.

File: src/components/common/Accordion/components/AccordionItem/AccordionItem.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
+import classes from "./AccordionItem.module.scss";
+
+interface AccordionItemProps {
+  className?: string;
+  itemTitle: string;
+  itemText: string;
+  itemId: number;
+  isItemOpen: boolean | undefined;
```
[the remark this answers is not in the document]
Comment (Austral1a) [reply src/components/common/Accordion/components/AccordionItem/AccordionItem.tsx:12]: "undefined" is left from the previous implementation, now it is gone.

**Pull request merged**